bbb-common-message/src/main/scala/org/bigbluebutton/common2/msgs/GroupChatMsg.scala

```diff
@@ -11,6 +11,7 @@ object GroupChatMessageType {
   val POLL = "poll"
   val BREAKOUTROOM_MOD_MSG = "breakoutRoomModeratorMsg"
   val PUBLIC_CHAT_HIST_CLEARED = "publicChatHistoryCleared"
+  val SYSTEM = "system"

```

Comment:
You should create a more specific message type `val USER_AWAY_STATUS_MSG = "userAwayStatusMsg"`.
This way the client can identify and apply the proper style for this type of message.

Reply:
I see. But akka-apps can't write messages because of the internationalization that is made in the bbb-html5 side.